ext/fg/js/frontend.js

```diff
@@ -149,6 +150,10 @@ class Frontend {
         }
     }
 
+    onMouseOut(e) {
+        this.popupTimerClear();
```

Comment:
I forgot to add a comment for this to the code, but it's actually intentionally clearing the timer even when mouse goes out of elements other than the parent iframe. Otherwise some timer that fires after the mouse leaves the parent popup can close the child popup because it doesn't find anything. Also worth mentioning that the timer is only used when scanning modifier is none.